components/authentication-framework/org.wso2.carbon.identity.application.authentication.framework/pom.xml

```diff
@@ -198,6 +192,22 @@
             <artifactId>slf4j-log4j12</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.graalvm.js</groupId>
+            <artifactId>js</artifactId>
+            <version>20.2.0</version>
```

Comment:
Remove the version from here and add to the dependency management section in the parent pom.